src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Manipulation_Tests.cs

```diff
@@ -27,6 +27,43 @@ public void TestManipulation()
 			TakeScreenshot("Result");
 		}
 
+		[Test] [AutoRetry] public void ManipulateDelta_DragLeft() => ManipulationDelta_Dragging(dx: 100);
+		[Test] [AutoRetry] public void ManipulateDelta_DragUp() => ManipulationDelta_Dragging(dy: 100);
+		[Test] [AutoRetry] public void ManipulateDelta_DragRight() => ManipulationDelta_Dragging(dx: -100);
+		[Test] [AutoRetry] public void ManipulateDelta_DragDown() => ManipulationDelta_Dragging(dy: -100);
+
+		public void ManipulationDelta_Dragging(int dx = 0, int dy = 0)
+		{
+			if (!(dx == 0 ^ dy == 0))
```

Comment:
A bit complex just to validate to validate if both are equals to 0?
```suggestion
			if (dx == 0 && dy == 0)
```

Reply:
more than that, "cannot be both 0 or both have value"